content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -127,8 +127,8 @@ DD_ENV=ci npm test
 Add `--require-module dd-trace/init` to however you normally run your `cucumber-js` tests, for example in your `package.json`:
 
 {{< code-block lang="javascript" filename="package.json" >}}
```

Comment:
Is `language="json"` an option here? (no idea, but if it is we should use that instead)

Reply:
oh yeah it seems there is a json option. The more you know